app/models/configuration.rb

```diff
@@ -0,0 +1,8 @@
+class Configuration < ApplicationRecord
+  validates :name, presence: true
+  validates :value, presence: true
+
+  def self.fetch(name)
+    Configuration.find_by(name: name).value
```

Comment:
We can drop the `Configuration.`

Reply:
We can make this safer - return `nil` if the config record doesn't exist.

We can make a `fetch!` as well if we need it later.